README.md

```diff
@@ -53,29 +55,34 @@ unzip booster-demo-front-spring-boot.zip
 cd booster-demo-front-spring-boot
 ```
 
-- Build, launch spring-boot locally to test if the front can be used in your browser
+- Build and launch spring-boot application locally to ensure the application can be used in your browser
 ```bash
 mvn clean spring-boot:run 
 open http://localhost:8090
 ```
-- Create a new OpenShift project on OCP
+
+- Deploy the application on the cloud platform using the `s2i` build process
 ```bash
 oc new-project cnd-demo
+oc new-app -f openshift/cloud-native-demo_front_template.yml
 ```
-- Deploy the application on the cloud platform using the `s2i` build process
+
+- Start the build using project's source
+  
 ```bash
-mvn package fabric8:deploy -Popenshift
```

Comment:
I prefer to keep fabric8:deploy for the front in order explain 2 different approaches which are similar to create the app on OpenShift. Can you keep it @geoand please

Reply:
Fixed :)